pipelining_example/pipelining_example.c

```diff
@@ -676,7 +676,7 @@ void fill_buffer(struct buffer *buf)
 
 	for (i = 0; i < block_num; i++) {
 		data += SERVER_DATA_SIZE;
-		*(uint64_t *)data = htonll(0x9ec65678f0debc9a);
+		*(uint64_t *)data = htonll(0xec7d5678f0debc9a);
```

Comment:
Let's add a comment here to clarify the hardcoded value. Something like this:
Since data and signature properties are currently hardcoded we pre-calculate T10DIF signature value. It is equal to 0xec7d5678f0debc9a which splits into CRC 0xec7d, APP_TAG 0x5678, REF_TAG 0xf0debc9a.